sdk/core/System.ClientModel/gen/ModelReaderWriterContextGenerator.Emitter.cs

```diff
@@ -31,6 +32,7 @@ private void EmitContextClass(ModelReaderWriterContextGenerationSpec contextGene
         {
             var contextName = contextGenerationSpec.Type.Name;
             var namespaces = GetNameSpaces(contextGenerationSpec);
+            namespaces.Add("System.Collections.Concurrent");
```

Comment:
Should this go in GetNamespaces?

Reply:
its only needed in the context file not the builder files so added it here only.